### PR TITLE
release updated dolly-v2-12b

### PR DESCRIPTION
**Pull request by choudharykartik**
import, evaluation run link: https://ml.azure.com/runs/sharp_basil_4h4ntxfh5v?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourcegroups/shared-finetuning-rg/workspaces/v-kartik2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

batch deployment run link: https://ml.azure.com/experiments/id/9a4f17fe-14f5-4965-b26d-f67f5109297e/runs/batchjob-174af73c-3aeb-4d2c-87fe-09669f529d47?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourcegroups/shared-finetuning-rg/workspaces/v-kartik2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

endpoint link: https://ml.azure.com/endpoints/realtime/dolly-12b-endpoint-2/test?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourcegroups/shared-finetuning-rg/workspaces/v-kartik2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

inference response:
image

ADO run link: https://dev.azure.com/msdata/Vienna/_build/results?buildId=111277843&view=results

conda.yaml
```
channels:
- conda-forge
dependencies:
- python=3.8.17
- pip<=23.2.1
- pip:
  - mlflow==2.6.0
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.6.0
  - azureml-core==1.53.0
  - azureml-mlflow==1.53.0
  - azureml-metrics[all]==0.0.26
  - scikit-learn==1.1.0
  - cryptography==41.0.3
  - python-dateutil==2.8.2
  - datasets==2.14.5
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.21.1
  - sentencepiece==0.1.99
  - transformers==4.33.1
  - torch==1.13.1
  - accelerate==0.23.0
  - Pillow==9.4.0
  - azureml-evaluate-mlflow==0.0.26
- pycocotools=2.0.4
name: mlflow-env

```

MLModel:
```
flavors:
  hftransformersv2:
    code: null
    config_hf_load_kwargs:
      trust_remote_code: true
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForCausalLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: databricks/dolly-v2-12b
    model_data: data
    model_hf_load_args:
      trust_remote_code: true
    pytorch_version: 1.13.1
    task_type: text-generation
    tokenizer_hf_load_kwargs:
      trust_remote_code: true
    transformers_version: 4.33.1
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.17
mlflow_version: 2.6.0
model_uuid: 187ac8ede80845b89f44f3cb7db47059
utc_time_created: '2023-11-21 07:09:11.845777
```

### CODE REVIEW

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 1]
```diff
@@ -2,20 +2,29 @@ $schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
 name: databricks-dolly-v2-12b
 path: ./
 properties:
-  SHA: 7dde7ac26ddcb679e0429fed1b6f751c1ffab1fb
+  SHA: 19308160448536e378e3db21a73a751579ee7fdd
   datasets: databricks/databricks-dolly-15k
   evaluation-min-sku-spec: 16|0|64|128
   evaluation-recommended-sku: Standard_NC24s_v3
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:8]: lets update this as well with list of comma sep. SKUs

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 2]
```diff
@@ -2,20 +2,29 @@ $schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
...
   inference-min-sku-spec: 0|24|448|1344
-  inference-recommended-sku: Standard_FX36mds, Standard_FX48mds, Standard_NC24s_v3,
-    Standard_ND96asr_v4, Standard_ND96amsr_A100_v4, Standard_ND40rs_v2
+  inference-recommended-sku: Standard_E16s_v3, Standard_FX36mds, Standard_FX48mds, Standard_NC24s_v3, Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:10]: Could you add these a100 computes as well for inference and evaluation alone: 

Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml (hunk 1 above)
Comment (choudharykartik) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml]: Done

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml (hunk 2 above)
Comment (choudharykartik) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml]: Added

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 3]
```diff
@@ -2,20 +2,35 @@ $schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
...
+  inference_compute_allow_list:
+  - Standard_E16s_v3
+  - Standard_FX36mds
+  - Standard_FX48mds
+  - Standard_NC24s_v3
+  - Standard_ND40rs_v2
+  - Standard_NC24ads_A100_v4
+  - Standard_NC48ads_A100_v4
+  - Standard_NC96ads_A100_v4
+  - Standard_ND96amsr_A100_v4
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:32]: Lets update description with updated inference results

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 4]
```diff
@@ -2,20 +2,35 @@ $schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
...
   languages: en
 tags:
   Featured: ''
   Preview: ''
-  computes_allow_list:
+  evaluation_computes_allow_list:
+  - Standard_NC24s_v3
   - Standard_ND40rs_v2
   - Standard_ND96asr_v4
+  - Standard_NC24ads_A100_v4
```
Comment (BhavanaTumma) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:19]: Are we only adding the ones we've tested or according to supported number of cores?

**Review by BhavanaTumma — APPROVED** [review]

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 5]
```diff
@@ -2,20 +2,35 @@ $schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
 name: databricks-dolly-v2-12b
 path: ./
 properties:
-  SHA: 7dde7ac26ddcb679e0429fed1b6f751c1ffab1fb
+  SHA: 19308160448536e378e3db21a73a751579ee7fdd
   datasets: databricks/databricks-dolly-15k
   evaluation-min-sku-spec: 16|0|64|128
-  evaluation-recommended-sku: Standard_NC24s_v3
+  evaluation-recommended-sku: Standard_NC24s_v3, Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4,Standard_ND96amsr_A100_v4, Standard_NC96ads_A100_v4
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:8]: please make sure its formatted correctly

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 6]
```diff
@@ -2,20 +2,35 @@ $schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
...
   inference-min-sku-spec: 0|24|448|1344
-  inference-recommended-sku: Standard_FX36mds, Standard_FX48mds, Standard_NC24s_v3,
-    Standard_ND96asr_v4, Standard_ND96amsr_A100_v4, Standard_ND40rs_v2
+  inference-recommended-sku: Standard_E16s_v3, Standard_FX36mds, Standard_FX48mds, Standard_NC24s_v3, Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4,Standard_ND96amsr_A100_v4, Standard_NC96ads_A100_v4
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:10]: please format it correctly

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml (hunk 4 above)
Comment (choudharykartik) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml:19]: No, we don't have tested it on Standard_NC24ads_A100_v4 due to quota issue but as this is working fine for Standard_NC24s_v3 so should work for higher computes also.

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml (hunk 3 above)
Comment (choudharykartik) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml:32]: updated

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml (hunk 6 above)
Comment (choudharykartik) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml]: updated

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml (hunk 5 above)
Comment (choudharykartik) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml]: updated

**Review by novaturient95 — APPROVED** [review]

**Review by BhavanaTumma — APPROVED** [review]

**Pull request merged**